### DISCUSSION TITLE
Add eye-catching warning about "(lazer) Running previous version may make all beatmaps and skins unreachable"

### DISCUSSION THREAD

**Discussion by str-str** [ppy/osu#17179] [Ideas] [answered]
I was doing mania mapping under 20220305-Release (latest) version, suffering from heavy cpu (or maybe gpu) load (resulting in the whole UI lagging in disappointing refresh rate whenever I scroll usingbroken) due to my complex timing and effects when switch from "Timing" to "composing"
(I forgetted why the new version was causing more resources)
My distribution is Gentoo, CPU and GPU are new enough (2 year newest) to afford heavy load.
![structure](https://user-images.githubusercontent.com/52829727/157308003-ef4a32ba-34c4-4fa5-8853-58586b2c73a6.png)
Using previous version of softwares is common if you are an open source software user.

This is the second time for me of losing beatmaps and skins in lazer, but exactly in the same case: running previous release version (Appimage) when have opened new version before. The first time was about 3 months ago when I was fresh to lazer

Now I have to seek methods of getting them back from SHA-256 encrypted folders and names

In all, to prevent people running previous versions **suffering from getting all files back**, in my opinion, "**adding eye-catching warning about this on release notification panel message ( or somewhere else in game client or github page** )  " is necessary

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/17158

**Reply by frenzibyte** [reply]
Also you don't really need to get them back from SHA-256 encrypted folders. You should be able to see `client_..._corrupt.realm` files generated from running older versions, renaming those back to `client.realm` while running the newer versions should bring back all your data.

This will all be better handled with the aforementioned PR anyway.
